expfmt/decode_test.go

```diff
@@ -22,10 +22,10 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/golang/protobuf/proto" //nolint:staticcheck // Ignore SA1019. Need to keep deprecated package for compatibility.
-	dto "github.com/prometheus/client_model/go"
-
 	"github.com/prometheus/common/model"
+
+	dto "github.com/prometheus/client_model/go"
+	"google.golang.org/protobuf/proto"
```

Comment:
Nit: In the Prometheus code base, we tend to keep imports from other repos than the current one in its own block _above_ the imports from the same repo (as it was before). And we usually also have the named imports in a different block (which wasn't adhered to before, but you could fix it now that we are on it).

Reply:
format is fixed.